pandas/tests/reshape/test_tile.py

```diff
@@ -37,6 +38,24 @@ def test_bins(self):
         tm.assert_almost_equal(bins, np.array([0.1905, 3.36666667,
                                                6.53333333, 9.7]))
 
+    def test_str_bins(self):
+        data = np.array([0.1, 0.1, 0.2, 0.5, 0.5, 0.9, 1.0])
```

Comment:
Add comment referencing the issue number above this.

Reply:
Addressed in f3323f4